### DISCUSSION TITLE
[Feature] Option to ask for help as beginner speaker

### DISCUSSION THREAD

**Discussion by vmaerten** [conference-hall/conference-hall#275] [Ideas and features]
Hello !
First of all, thanks for the tool and the new version!

# Description
Hello, I'm part of the TouraineTech team and we want to encourage beginner speakers to submit their talk ideas. The idea behind is to offer them concrete help during the preparation and the rehearsal of their talks.

We would like to be able to add a checkbox (that we can enable/disable in the event configuration) that the speaker can check to ask for help, something like:

> 
> [x] I am a beginner speaker and would like some help during the preparation of my talk
>

I think this could really benefit the tech world in general: this kind of help is proposed in more and more events (Young Blood, Tremplin des Speakers, ...) and it seems to be a good way to improve the diversity situation in our field.

We currently handle it by creating specific format ("Conférence Débutant·e", etc) but it's not very optimal.

(Originally posted by Jordan [here](https://github.com/bpetetot/conference-hall-beta/issues/937))

if you still think it's a good idea, I might give this a try 🙂

**Comment by bpetetot** [comment]
Thanks for the idea.

We start working on "custom survey for speakers". You will be able to ask any questions to speakers when they submit a talk and get access to the answers in proposal pages.

The custom survey will allow to ask 3 types of questions: free text, multi-choice (checkbox), single-choice (radio). You can setup a label, a description to each question. A question can be required or optional.

I think this feature will fit with your needs. It should be available in the coming weeks 😉

**Reply by bpetetot** [reply]
Customized surveys are now available. You can ask the speaker up to 8 questions before submitting a talk. (questions can be required or optional)

**In the event settings:**
![image](https://github.com/user-attachments/assets/99e2f8ba-8ba8-49ff-b0f3-55b0b4929a36)

**Asked to the speaker before the submission:**
![image](https://github.com/user-attachments/assets/abc9d474-21cc-4db0-bc4f-876bf28b87ff)
